### PR TITLE
Bump compatible version to WP 6.2

### PR DESCRIPTION
**Pull request by ggutenberg**
### Description of the Change

Partial testing has been done against WP 6.2:
- `npm run test` is passing.
- `npm run test -c tests/phpunit/multisite.xml` is passing.
- `npm run test --group [ajax|ms-files|external-http]` are all erroring out with `Cannot open file "[group name]"`.
- `npm run tests:e2e` is erroring out with `./vendor/bin/wpacceptance: not found`.
- Cypress tests are all passing.

I manually tested ChatGPT excerpt generation which is working well.
I'm unable to create an Azure account as they don't accept VoIP numbers for account validation, and my billing country doesn't match my non-VoIP phone number. IBM Cloud had a similar issue.

I recommend that additional testing be done against Azure and Watson before this PR is merged.

Changes in this PR include bumping the compatible version to WP 6.2 and adding a `.nvmrc` pointing at latest LTS (18.15.0).

### How to test the Change

To test PHPUnit, edit `bin/install-wp-tests.sh` and add the following on lines 50-51:

```bash
WP_TESTS_TAG="trunk"
WP_VERSION="6.2-RC4"
```

Then run:

```bash
npm run install_tests
npm run test
```

To test Cypress against WP 6.2 do the following:

Create a file called `.wp-env.override.json` in the root of this folder with the following contents:
```json
{
  "core": "WordPress/WordPress#6.2-branch"
}
```

Then run the following commands:
```bash
nvm install
npm ci
npm run build
npm run env start
npm run env clean all
npm run postenv:start
npm run cypress:run
```

### Changelog Entry

> Changed - Bump "Tested up to" compatibility to WP 6.2.

### Credits

Props @ggutenberg 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18.15.0
```
Comment (ravinderk) [inline .nvmrc]: @ggutenberg This change is out of scope. Can we remove it? And open another pull request to upgrade node to `v18.15`

I think there are other places where like
- We should change the node versions like Github workflows.
- We should also check does it impact the package.lock file version.

File: readme.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Contributors:      10up, jeffpaul, dkotter
 Tags:              AI, Artifical Intelligence, ML, Machine Learning, Microsoft Azure, IBM Watson, OpenAI, ChatGPT, Content Tagging, Classification, Smart Cropping, Alt Text
 Requires at least: 5.7
-Tested up to:      6.1
+Tested up to:      6.2
```
Comment (ravinderk) [inline readme.txt:5]: @ggutenberg Can we check other places like `readme.txt`, conditional changes in code, etc, and upgrade them too?

**Review by ravinderk — COMMENTED** [review]
@ggutenberg, I left a few quick comments. 

Feel free to ask any questions

File: .nvmrc (hunk 1 above)
Comment (ggutenberg) [reply .nvmrc]: No problem. I didn't see any indication of node version in `package.json` though, so I wasn't sure which node version this was supposed to work with.

File: readme.txt (hunk 2 above)
Comment (ggutenberg) [reply readme.txt:5]: I've bumped the badge in `README.md` to 6.2, but I don't see any other references to 6.1 as the WP version in the codebase. Is there a specific conditional you're aware of that I can take a look at to see what it's checking for?

Comment (ravinderk) [reply readme.txt:5]: @ggutenberg I overlooked `readme.txt`, so we are good on this.

-----------

I did not see this information in the plugin header (main plugin file `classifai.php`). Can you add this?

You can read more about plugin header requirements here: https://developer.wordpress.org/plugins/plugin-basics/header-requirements/

Comment (ggutenberg) [reply readme.txt:5]: [Add minimum WP and PHP versions to plugin header](https://github.com/10up/classifai/pull/420/commits/1afb05952c3a03be1119fa226530cc875e2aa6a5)

There doesn't seem to be an option for "Tested up to" in the header.

**Review by ravinderk — APPROVED** [review]
Looks good.
